utils/utils.go

```diff
@@ -91,9 +93,23 @@ func OSRelease(key string, file ...string) (string, error) {
 		// We try with the old naming without the prefix in case the key wasn't found
 		v, exists = release[key]
 		if !exists {
-			return "", KeyNotFoundErr{Err: fmt.Errorf("%s key not found", kairosKey)}
+			// We try with fallback file
+			release, err = godotenv.Read(osReleaseFallback)
+			if err != nil {
+				return "", err
+			}
+			kairosKey = "KAIROS_" + key
+			v, exists = release[kairosKey]
+			if !exists {
+				// We try with the old naming without the prefix in case the key wasn't found
```

Comment:
It's been a long time since we introduced the prefix. What is this trying to catch?

Reply:
really old releases upgrading to newer ones